src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/BrowseWfsLayers/BrowseWfsLayers.xaml.cs

```diff
@@ -0,0 +1,149 @@
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Ogc;
+using Esri.ArcGISRuntime.Symbology;
+using System;
+using System.Diagnostics;
+using System.Drawing;
+using System.Linq;
+using System.Windows;
+
+namespace ArcGISRuntime.WPF.Samples.BrowseWfsLayers
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Browse a WFS service for layers",
+        "Layers",
+        "Browse for layers in a WFS service.",
+        "")]
+    public partial class BrowseWfsLayers
+    {
+        // URL to the WFS service.
+        private const string ServiceUrl = "https://dservices2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/services/Seattle_Downtown_Features/WFSServer?service=wfs&request=getcapabilities";
+
+        public BrowseWfsLayers()
+        {
+            InitializeComponent();
+            Initialize();
+        }
+
+        private async void Initialize()
+        {
+            // Create the map with imagery basemap.
+            MyMapView.Map = new Map(Basemap.CreateImagery());
+
+            // Create the service.
+            WfsService service = new WfsService(new Uri(ServiceUrl));
+
+            // Load the service.
+            await service.LoadAsync();
+
+            // Show the layers in the UI.
+            WfsLayerList.ItemsSource = service.ServiceInfo.LayerInfos.Reverse();
+
+            // Update the UI.
+            LoadingProgressBar.Visibility = Visibility.Collapsed;
+            LoadLayersButton.IsEnabled = true;
+        }
+
+        private async void LoadLayers_Clicked(object sender, RoutedEventArgs e)
+        {
+            // Skip if nothing to add.
+            if (WfsLayerList.SelectedIndex == -1)
+            {
+                return;
+            }
+
+            // Show the progress bar and hide the button.
+            LoadingProgressBar.Visibility = Visibility.Visible;
+            LoadLayersButton.IsEnabled = false;
+
+            // Clear the existing layers.
+            MyMapView.Map.OperationalLayers.Clear();
+
+            try
+            {
+                // Get the selected layer.
+                WfsLayerInfo selectedLayerInfo = (WfsLayerInfo)WfsLayerList.SelectedItems[0];
+
+                // Create the feature table.
+                WfsFeatureTable table = new WfsFeatureTable(selectedLayerInfo);
+                
+                // Set the table's feature request mode.
+                table.FeatureRequestMode = FeatureRequestMode.ManualCache;
+
+                // Set the axis order based on the UI.
```

Comment:
Coordinate switching affects all geometries. Some services use X,Y, while others use Lat,Long, which maps to Y,X. In this case, Runtime guesses correctly and it will do so for most services.

I include the option in this sample because the developer should expose it whenever the service or layer isn't known ahead of time. Apps like QGIS and ArcGIS Pro have to expose it because they can't always get it right either. 

The API ref doc was recently reviewed and expanded to clarify this issue. It is also covered in the guide doc.